plutus-ledger-api/src/PlutusLedgerApi/V3/ParamName.hs

```diff
@@ -97,6 +97,12 @@ data ParamName =
   | FstPair'memory'arguments
   | HeadList'cpu'arguments
   | HeadList'memory'arguments
+  | LengthArray'cpu'arguments
```

Comment:
New parameters must be appended, not inserted in the middle.

And these names don't seem accurate - e.g., `ListToArray` is linear, so there needs to be an intercept and a slope. If they aren't accurate, then I'm not sure what the point is adding them at all. @kwxm What do you think?

Reply:
Also these need to be added to V1 and V2 as well.

Reply:
> I wrote a golden test which will fail if this condition is violated

How do we update the golden file when new parameters are added?

> The point is to make build green. Some parameters have to be added, otherwise CI tests fail.

I'll let @kwxm check whether this is the right thing to do.

Reply:
> How do we update the golden file when new parameters are added?

`cabal test plutus-ledger-api-test`  fails if new parameter names were inserted in the middle, not appended.
`cabal test plutus-ledger-api-test --test-options --accept` updates the golden file.

Reply:
The easiest thing to do would have been to update `plutus-tx` and `plutus-tx-plugin` and deal with `plutus-ledger-api` in a separate PR after the costing gets done.  I think it'd all get through CI in that case, although I might be wrong.